pkg/pgsql/server/session_factory.go

```diff
@@ -17,34 +17,24 @@ limitations under the License.
 package server
```

Comment:
_**pkg/server/databaselist.go**_ being moved and renamed to `pkg/pgsql/server/session_factory.go` looks a bit weird, doesn't it? Why would something that has a general purpose to something that is pgsql specific?
Or perhaps it's just git diff making it look weird?

Reply:
I think it's the second. Databaselist is something I moved from immudb server package to immudb database package